website/src/components/Footer.tsx

```diff
@@ -1,40 +1,27 @@
-import { useColorMode } from "@chakra-ui/react";
+import { Box, Divider } from "@chakra-ui/react";
 import Image from "next/image";
 import Link from "next/link";
 import { useMemo } from "react";
 
 export function Footer() {
-  const { colorMode } = useColorMode();
-  const bgColorClass = colorMode === "light" ? "bg-transparent" : "bg-gray-800";
-  const borderClass = colorMode === "light" ? "border-slate-200" : "border-transparent";
-
   return (
-    <footer className={bgColorClass}>
-      <div className={`flex mx-auto max-w-7xl justify-between border-t p-10 ${borderClass}`}>
-        <div className="flex items-center pr-8">
-          <Link href="/" aria-label="Home" className="flex items-center">
-            <Image src="/images/logos/logo.svg" className="mx-auto object-fill" width="52" height="52" alt="logo" />
+    <footer>
```

Comment:
This looks like it changes the footer for all views.  I actually like the footer for the landing page and for most other views.  If we're changing this, can we make a new component `SlimFooter`or somehow structure this so it renders differently depending on props?

Reply:
Sure thing!